pennylane/pulse/__init__.py

```diff
@@ -68,6 +68,7 @@
     :toctree: api
 
     ~rydberg_interaction
+    ~transmon_interaction
     ~drive
```

Comment:
Is the plan to still expose `drive` to users or only `rydberg_drive` and `transmon_drive`?

Reply:
All drives would be exposed :)